app/views/drafts/shared/_javascript.html.erb

```diff
@@ -1,7 +1,4 @@
 <script>
   // Load json-schema for field validation
   var globalJsonSchema = <%= raw @schema.parsed_json.to_json %>;
-
-  // Setup Science Keywords
-  setupScienceKeywords(<%= raw @science_keywords.to_json %>, { field_prefix: 'variable_draft/draft' });
```

Comment:
is there a reason this was moved from the partial to the edit view?

Reply:
The original version would have had it loading it on new forms, but I didn't want to do that so I moved it to the edit partial. I have conditionals in it though, I'll put it back